internal/terraform/lang/schema.go

```diff
@@ -69,18 +69,21 @@ func schemaAttributeDetail(attr *tfjson.SchemaAttribute) string {
 		requiredText = "Required"
 	}
 
-	return strings.TrimSpace(fmt.Sprintf("(%s, %s) %s",
-		requiredText, attr.AttributeType.FriendlyName(), attr.Description))
+	return strings.TrimSpace(fmt.Sprintf("%s, %s",
+		requiredText, attr.AttributeType.FriendlyName()))
 }
 
 func schemaBlockDetail(blockType *BlockType) string {
 	blockS := blockType.Schema()
 
-	requiredText := "Required"
-	if len(blockType.BlockList) >= int(blockS.MinItems) {
-		requiredText = "Optional"
+	detail := fmt.Sprintf("Block, %s", blockS.NestingMode)
```

Comment:
I reckon we'll iterate on the human-readable text here over time to e.g. print `list of blocks`, `single block` etc. but this is already big step forward and I like these changes 👍 💯

Reply:
Agree, just wanted to move closer to core concepts.